src/components/MajorList/index.js

```diff
@@ -36,13 +36,13 @@ export default class Majors extends Component {
     }
 
     // Creates the list item's title
-    if (this.props.majors.length === 1) {
+    if (this.props.majors.length < 2) {
       majorPrefix = (
```

Comment:
This has been updated so that if a person doesn't have any majors, the Major list item will still appear.